File: src/main/java/dto/BookingDTO.java
package dto;

import lombok.*;

import java.time.LocalDateTime;
import java.util.List;
import entity.Passenger;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
public class BookingDTO {
    private Integer bookingId;
    private LocalDateTime bookingDate;
    private Integer noOfPax;
    private Double tourAmount;
    private Double taxes;
    private Double totalAmount;
    private String paymentStatus;
    private Integer customerId;
    private Integer packageId;
    private Integer departureId;
    private List<PassengerDTO> passengers;
    
}

File: DotNet_Backend/SafarnamaApplication/SafarnamaApplication/DTOs/BookingDTO.cs
namespace SafarnamaApplication.DTOs
{
    public class BookingDTO
    {
        public int BookingId { get; set; }
        public DateTime BookingDate { get; set; }
        public int NoOfPax { get; set; }
        public double TourAmount { get; set; }
        public double Taxes { get; set; }
        public double TotalAmount { get; set; }
        public string? PaymentStatus { get; set; }
        public int? CustomerId { get; set; }
        public int? PackageId { get; set; }
        public int? DepartureId { get; set; }
        public List<PassengerDTO>? Passengers { get; set; }
    }
}
